tests/run-tests.sh

```
@@ -99,6 +99,67 @@ install_dependencies() {
+        return 1
+    fi
+
+    # Extract payload (second part between dots)
+    local payload=$(echo "$token" | cut -d'.' -f2)
+
+    if [ -z "$payload" ]; then
+        return 1
+    fi
+
+    # Decode payload
+    local decoded=$(base64url_decode "$payload" | base64 -d 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)